src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -680,10 +680,8 @@ public Object onTraversalSuccess() {
 
                 if (outputModel != null) {
                     final GATKReport report = writeModelReport(goodModel, badModel, USE_ANNOTATIONS);
-                    try (final PrintStream modelReportStream = new PrintStream(outputModel)) {
+                    try (final PrintStream modelReportStream = new PrintStream(outputModel.getOutputStream())) {
                         report.print(modelReportStream);
-                    } catch (FileNotFoundException e) {
-                        throw new UserException.CouldNotCreateOutputFile("File: (" + outputModel + ")", e);
```

Comment:
Handle errors on the `PrintStream`

Reply:
Added a local checkError call, same as above.